sklearn/utils/testing.py

```diff
@@ -796,6 +796,20 @@ def _get_args(function, varargs=False):
         return args
 
 
+def _get_parent_args(Estimator):
+    """Helper to get list of parents properties"""
+    # NOTE this works only in python3.5
+    if sys.version_info < (3, 5):
+        NotImplementedError("_get_parent_args is not available "
```

Comment:
I'm not happy about having py3-only estimator checks. @raghavrv, what is the reason `_get_args` could not use Py2 insoect?